front/package.json

```diff
@@ -11,6 +11,10 @@
     "format": "prettier --write \"src/**/*.{ts,tsx}\"",
     "prepare": "cd .. && husky install front/.husky"
   },
+  "headwind.classRegex": {
+    "html": "\\bclass\\s*=\\s*[\\\"\\'](https://github.com/heybourn/headwind/blob/master/[_a-zA-Z0-9\\s\\-\\:\\/]+)[\\\"\\']",
+    "javascriptreact": "(?:\\bclassName\\s*=\\s*[\\\"\\'](https://github.com/heybourn/headwind/blob/master/[_a-zA-Z0-9\\s\\-\\:\\/]+)[\\\"\\'])|(?:\\btw\\s*`([_a-zA-Z0-9\\s\\-\\:\\/]*)`)"
+  },
```

Comment:
what is this? is like a prettier to tailwind?

Reply:
yep